_data/simple-icons.json

```
@@ -8827,8 +8827,8 @@
         },
         {
             "title": "Lerna",
-            "hex": "334155",
-            "source": "https://github.com/lerna/website/blob/dcef9c248b2b67bdd2214058beca437732690226/images/lerna-logo-dark.svg"
+            "hex": "9333EA",
```

```suggestion
            "hex": "C084FC",
```

The color of the favicon is a darker purple which I'm guessing is what you used here, but on the website itself the primary purple used is `#c084fc`. Shall we use that instead?

---

Sounds good. Let's use the more colorful one.